### DISCUSSION TITLE
Does BlueSCSI 2 support CDROM devices with 512 byte blocks?

### DISCUSSION THREAD

**Discussion by dburr** [BlueSCSI/BlueSCSI-v2#87] [Help] [answered]
Subject says it all. Most Sun workstations' firmware expect CDROM devices to have a blocksize of 512 bytes, and will not boot otherwise. Iirc I tried this some time ago, put a bootable Solaris ISO on a SD card and named it "CD60_512.iso" but was not able to get it to work (either the file was completely ignored and/or it otherwise did not work.) But it's been a while and I may be misremembering.

**Comment by erichelgeson** [comment, marked as answer]
Yes it does - I've seen people report it working in Discord - though I have no Sun workstation to try it on. If there's a debug log you can include on the issue you had we can take a look at it. Please also make sure you're on the last firmware.

**Reply by dburr** [reply]
Thanks! Just tried it again after upgrading to the latest firmware and all is working perfectly now, NetBSD is installing as we speak (type?)
